nh_eobs_mental_health/tests/common/transaction_observation.py

```diff
@@ -28,21 +28,13 @@ def setUp(self):
         self.expected_risk = None
         self.expected_freq = None
```

Comment:
Let's discuss as I think you've got your own test setup stuff similar to this anyway